lib/clean_architecture/serializers/success_object_payload.rb

```diff
@@ -5,23 +5,26 @@
 
 module CleanArchitecture
   module Serializers
-    class SuccessPayload
+    class SuccessObjectPayload
```

Comment:
What's the thinking behind this name change?

Reply:
The primary reason was consistency. The other example in the codebase is `TargetedParameters` and `UntargetedParameters`. I felt after having added the `SuccessCollectionPayload` class,  `SuccessPayload` wasn't enough to convey the distinction between the two.

Reply:
OK - if that's the case I think we should see if we can come up with a better option, or someone can convince me that `Object` isn't as meaningless as I think it is.

Reply:
`Object` was just to connote that its success payload for a single object (entity) as opposed to the other collection success payload. Will `SingleEntitySuccessPayload` be a better name?

Reply:
Unfortunately I think that implies it gives back an 'Entity', whereas its just a Value within a kind of 'Success' wrapper. I actually think just `SuccessPayload` is fairly clear, you'll got `SuccessPayload` singular, `SuccessCollectionPayload` for multiple?

Reply:
That's good to know, because I was thinking of it like `SingleEntitySuccessPayload` will connote, it's a payload (can be whatever format we decided for the payload, so it's what's inside agnostic), and `SingleEntity` will mean it has a `Single Entity` of some kind inside. That coupled with the `Serializers` namespace will mean `it's a payload with a single entity serialized in some way inside`.

I problem I have with just `SuccessPayload` is, then the name `SuccessPayload` will have no connotation of that it's meant to be used for a single entity (and there's a different one for collection), we could end up using it like https://github.com/tricycle/bellroy-siegfried/blob/master/lib/payments/entities/hourly_statistics.rb again where the response will not be JSON-api compliant.

But that risk lies with all other things, so I am not too fussed about it. (And I need it merged asap to make required changes in my branch and finish that story)